plutus-core/plutus-ir/src/PlutusIR/Transform/Inline.hs

```diff
@@ -116,7 +112,7 @@ inline. This is essentially the reason for the existence of the UPLC inlining pa
 -}
 
 -- | Substitution range, 'SubstRng' in the paper.
--- No 'Susp' case because we don't do PreInlineConditionally.
+-- IS THIS STILL RIGHT? No 'Susp' case because we don't do PreInlineConditionally.
```

Comment:
I think it's wrong: we do do `PreInlineUnconditional`, but with the tweak from my comment. So we don't need `Susp`, still.